samples/deployment/vmware/deploy_and_destroy.yaml

```diff
@@ -0,0 +1,73 @@
+- name: "Deploy and destroy vFTD in the VMware cloud"
+  hosts: localhost
+  gather_facts: no
+  vars_files:
+    - ./vars.yaml
+  tasks:
+    - name: "Download OVF"
```

Comment:
Please add support to check of OVF file existence to prevent re-downloading.

Reply:
fixed